src/components/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios"
+
+export function fetchArticleData() {
+    return axios
+    .get("https://be-nc-news-project.onrender.com/api/articles")
+    .catch((err) => console.log(err))
```

Comment:
Catch blocks should be at the end of a promise chain. In this case that would be in the file in which fetchArticleData was invoked